src/libsync/jobqueue.cpp

```diff
@@ -36,9 +36,11 @@ void JobQueue::block()
 
 void JobQueue::unblock()
 {
+    if (!_blocked) {
```

Comment:
As `_blocked` is an unsigned int, not a bool, I'd rather use an explicit comparison against 0.